profiling/nvtx-connector/kp_nvtx_connector.cpp

```diff
@@ -23,12 +23,20 @@
 
 #include "kp_core.hpp"
 
+static int tool_globfences;  // use an integer
+
 namespace KokkosTools {
-namespace NVProfConnector {
+namespace NVTXConnector {
 
 void kokkosp_request_tool_settings(const uint32_t,
                                    Kokkos_Tools_ToolSettings* settings) {
-  settings->requires_global_fencing = false;
+  settings->requires_global_fencing = true;
```

Comment:
without this kernel renaming doesn't work, and the regions all only measure kernel launch